### PR TITLE
more info to run user-cluster-controller-manager locally

### PR DESCRIPTION
**Pull request by SimonTheLeg**
Signed-off-by: Simon Bein <simontheleg@gmail.com>

**What this PR does / why we need it**:

I think it is a good time to make the user-cluster-controller-manager part of the guide a bit more specific to help new-joiners.

**What type of PR is this?**

/kind documentation

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
none
```

**Documentation**:

```documentation
none
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,13 +78,27 @@ cd kubermatic
...
 
-#### Running locally
+##### user-cluster-controller-manager
+
+In order to instrument the seed-controller to allow for a local user-cluster-controller-manager, you need to add a `worker-name` label with your local machine's name as its value. Additionally, you need to scale down the already running deployment.
+
+```sh
+# Using a kubeconfig, which points to the seed-cluster
+export cluster_id="<id-of-your-user-cluster>"
+kubectl patch cluster ${cluster_id} --type=merge --patch "{\"metadata\":{\"labels\":{\"worker-name\":\"$(uname -n)\"}}}"
```
Comment (embik) [inline README.md]: Maybe simplify this to `kubectl label cluster ${cluster_id} worker-name=$(uname -n)`? :D

Comment (SimonTheLeg) [reply README.md]: of course kubectl has its own subcommand for labelling 🙈 :D 
Will update the guide :)

File: README.md [hunk 2]
```diff
@@ -78,13 +78,27 @@ cd kubermatic
 ```
 
 There are a couple of scripts in the `hacks` directory to aid in running the components locally
-for testing purposes.
+for testing purposes. You can create a cluster via the UI at `https://dev.kubermatic.io`.
```
Comment (xrstf) [inline README.md]: I feel uncomfortable to bring back all those references to our dev system into this public README. This sentence would only hold true for Kubermatic employees and so I don't think it belongs here. In general we are moving away from relying on dev for everything.

Comment (SimonTheLeg) [reply README.md]: I see where you are coming from. Removed the ref.

**Review by embik — APPROVED** [review]

**Pull request merged**